biosys/apps/main/api/serializers.py

```diff
@@ -32,8 +32,8 @@ class UsernameValidator(RegexValidator):
     A username validator that allows `\` (backslash). This is a requirement imposed by NSW OEH.
     By default Django and DRF doesn't allow it.
     """
-    regex = r'^[\w.@+-\\]+$'
-    message = 'Enter a valid username. This value may contain only letters, numbers, and @/./+/-/_/\ characters.'
+    regex = r'^[a-z0-9@+\\.-]+$'
+    message = 'Enter a valid username. This value may contain only lowercase letters, numbers, and @/./+/-/_/\ characters.'
```

Comment:
@serge-gaia i could see a test case for this... but i don't see why it would work based on this:
https://regex101.com/r/W9YVOz/1

Reply:
Could you create a test case

Reply:
This is just for the serialiser (API) what about the web form?

Reply:
ok, can make one, also, wouldn't the app + webform still be using this api?